src/components/BotProfileImage.tsx

```diff
@@ -1,7 +1,7 @@
 import styled from 'styled-components';
 
 import { getColorBasedOnSaturation } from '../colors';
-import { ReactComponent as Icon } from '../icons/bot-profile-image-small.svg';
```

Comment:
Why this format have to be changed? 
- Verbose to use
- It causes a conflict with UIKit svg import type and throwing an error 🥲